app/models/policies/international_relocation_payments/admin_tasks_presenter.rb

```diff
@@ -9,6 +9,12 @@ def initialize(claim)
         @claim = claim
       end
 
+      def arrival_date
+        [
+          ["Arrival date", eligibility.date_of_entry&.to_fs]
```

Comment:
```suggestion
          ["Arrival date", eligibility.date_of_entry.to_fs]
```
I think we can assume all IRP eligibilities will have a date of entry as it's required to complete the journey